test/system/tests/print-files/quote.printed

```diff
@@ -0,0 +1,20 @@
+magic: ff544347
```

Comment:
This and the binary blob should go under test fixtures: test/system/fixtures.

However, the test is missing that we want to detect if/when the binary blob output changes and breaks print. I would just output that blob for the test, and run print on it, and use a python script to load the yaml and verify what you can, ie that certain fields exist, etc.

Reply:
Sure, I can add a case to grab a quote and validate it. However, I'd like to have at least one premade blob with known good value -- I.e. a quote blob who's digest was validated by hand in a hexdump.

Reply:
That sounds good.